### ISSUE TITLE
npmignore remove babelre

### ISSUE THREAD

**Issue by sheepwithouthorn** [gaearon/disposables#9]
npmignore remove babelre please!

**Comment by nanjixiong218** [comment]
+1

**Comment by JennerChen** [comment]
+1

**Comment by Tankpt** [comment]
pls

**Comment by PDS42** [comment]
Workaround that did the trick for me: add `exclude: /(disposables)/` in your babel-loader webpack config, for instance:
```
{
     test: /\.js$/,
    exclude: /(disposables)/,
    loader: 'babel-loader',
}
```

**Comment by gaearon** [comment]
If you have this problem it means your Babel setup is incorrect and you're mistakingly compiling everything in `node_modules`. Not only this is slow, but it also can mess up the code.

**Comment by gaearon** [comment]
That said I published a "fix" in `disposables@1.0.2`.
